runtime/doc/diagnostic.txt

```diff
@@ -373,39 +373,8 @@ config({opts}, {namespace})                          *vim.diagnostic.config()*
                                      Otherwise, all signs use the same
                                      priority.
 
-                                 • float: Options for floating windows:
-                                   • severity: See |diagnostic-severity|.
-                                   • header: (string or table) String to use
-                                     as the header for the floating window. If
-                                     a table, it is interpreted as a [text,
-                                     hl_group] tuple. Defaults to
-                                     "Diagnostics:".
-                                   • source: (string) Include the diagnostic
-                                     source in the message. One of "always" or
-                                     "if_many".
-                                   • format: (function) A function that takes
-                                     a diagnostic as input and returns a
-                                     string. The return value is the text used
-                                     to display the diagnostic.
-                                   • prefix: (function, string, or table)
-                                     Prefix each diagnostic in the floating
-                                     window. If a function, it must have the
-                                     signature (diagnostic, i, total) ->
-                                     (string, string), where {i} is the index
-                                     of the diagnostic being evaluated and
-                                     {total} is the total number of
-                                     diagnostics displayed in the window. The
-                                     function should return a string which is
-                                     prepended to each diagnostic in the
-                                     window as well as an (optional) highlight
-                                     group which will be used to highlight the
-                                     prefix. If {prefix} is a table, it is
-                                     interpreted as a [text, hl_group] tuple
-                                     as in |nvim_echo()|; otherwise, if
-                                     {prefix} is a string, it is prepended to
-                                     each diagnostic in the window with no
-                                     highlight.
-
+                                 • float: Options for floating windows. See
```

Comment:
Is this correct/up-to-date? Seems like a big deletion

Reply:
See https://github.com/neovim/neovim/pull/16105